cmd/cli/reconcile_resource.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2025 Stefan Prodan.
+// SPDX-License-Identifier: AGPL-3.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/fluxcd/pkg/apis/meta"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var reconcileResourceCmd = &cobra.Command{
+	Use:   "resource",
+	Short: "Trigger Flux resource reconciliation",
+	Example: `  # Trigger the reconciliation of a Flux Kustomization
+  flux-operator -n apps reconcile resource Kustomization/my-app
+
+  # Force reconcile a Flux HelmRelease
+  flux-operator -n apps reconcile resource HelmRelease/my-release --force
+
+  # Trigger the reconciliation of an OCIRepository without waiting for it to become ready
+  flux-operator -n apps reconcile resource OCIRepository/my-app --wait=false
+`,
+	Args:              cobra.ExactArgs(1),
+	RunE:              reconcileResourceCmdRun,
+	ValidArgsFunction: resourceKindCompletionFunc(),
+}
+
+type reconcileResourceFlags struct {
+	force bool
```

Comment:
`HelmRelease` also supports the `--reset` flag in `flux reconcile hr` (I think it's the only exception missing)